gptme/prompts.py

```diff
@@ -247,6 +247,18 @@ def get_system_distro() -> str:
     return "Linux"
 
 
+def get_system_distro_version() -> str:
```

Comment:
I would merge this with `get_system_distro` and make it return a `tuple[str, str]`